.gitlab/ci/.gitlab-ci.on-push.yml

```diff
@@ -760,6 +762,15 @@ test-upload-flow:
     - python3 ./Utils/test_upload_flow/verify_bucket.py -a "${ARTIFACTS_FOLDER}" -s $GCS_MARKET_KEY -sb $current_storage_base_path -b $ALL_BUCKETS
     - section_end "Verify Created Testing Bucket"
     - job-done
+  after_script:
```

Comment:
running in after_script to delete the branch regardless if the previous step succeeded.